src/caveexpress/server/entities/Water.cpp

```diff
@@ -11,13 +11,18 @@
 
 namespace caveexpress {
 
+const float min_lastSoundDT = 1.f;  // time, sec
```

Comment:
`_lastSoundTime` exists for this - why do we need a global var here?

Reply:
> `_lastSoundTime` exists for this - why do we need a global var here?

This is a const value for comparing later:
`	if (_lastSoundDT > min_lastSoundDT) {`
while `_lastSoundTime` is variable, added in `Water::update` by `deltaTime`.

Reply:
i see - but why do we need `_lastSoundDT` if we have `_lastSoundTime` (i don't see why it must be float, we should be able to re-use the existing var, no?)